webapp/src/utils.test.ts

```diff
@@ -206,7 +206,7 @@ describe('utils', () => {
             const sleepTimeMs = 500;
             const start = Date.now();
             await sleep(sleepTimeMs);
-            expect(Date.now() - start).toBeGreaterThan(sleepTimeMs);
+            expect(Date.now() - start).toBeGreaterThanOrEqual(sleepTimeMs);
```

Comment:
This kept failing, so I assume the change is ok? What a weird test though -- why do we have it?

Reply:
Well, I just added that as part of the retry logic. Just testing that sleeping actually sleeps for at least the given duration which could be equal I guess so it's a proper fix.